deepmd/utils/argcheck.py

```diff
@@ -503,11 +503,11 @@ def descrpt_se_atten_args():
             doc=doc_only_tf_supported + doc_stripped_type_embedding,
         ),
         Argument(
-            "smooth_type_embdding",
+            "smooth_type_embedding",
```

Comment:
We may keep the original one as an alias, so there's no breaking change.

Reply:
+1